### ISSUE TITLE
ProfileDrawerItem info not updated

### ISSUE THREAD

**Issue by androideveloper** [mikepenz/MaterialDrawer#2179]
## About this issue

After upgrade from 5.9.5 to 6.0.2 I have found another issue. I have MiniDrawer  with ProfileDrawerItem. When I have new data for ProfileItem, I call
`        ProfileDrawerItem activeProfile = (ProfileDrawerItem) mAccountHeader.getActiveProfile();
        activeProfile
                .withName(salesAgent != null ? salesAgent.getName() : getString(R.string.sales_agent))
                .withEmail(Params.getCompanyName());
        mAccountHeader.updateProfile(activeProfile);`
But it doesn't update profile data, only after activity recreate. Maybe I need to call some additional code to update profile?
The issue is present only after upgrade.

## Details
- [ ] Library 6.0.2
- [ ] Support library 27.0.2
- [ ] Build Tools 27.0.3
- [ ] AS 3.0.1

**Comment by mikepenz** [comment]
your profile does not have a identifier?

**Comment by androideveloper** [comment]
Yes. But in previous version it was auto-generated why you removed that code?

**Comment by mikepenz** [comment]
does it work if you provide an identifier?

**Comment by androideveloper** [comment]
Yes

**Comment by mikepenz** [comment]
This will be fixed in the new udpate
